src/domain/application/components/ApplicationContacts.tsx

```diff
@@ -155,6 +185,11 @@ const CustomerFields: React.FC<{
           autoComplete="on"
           defaultValue={null}
           required={applicationType === 'EXCAVATION_NOTIFICATION' && registryKeyInputEnabled}
+          helperText={
+            isRegistryKeyHiddenNotificationShown
```

Comment:
Displaying helper text could be determined straight from `registryKeyHidden` without the need for `useState` and `useEffect` for that.

Reply:
Good point, thanks. Changed it.